addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from odoo.exceptions import ValidationError
 from odoo import api, fields, models, _
```

Comment:
This import should be the second one (sorted)
https://www.odoo.com/documentation/14.0/fr/developer/misc/other/guidelines.html#imports
```suggestion
from odoo import api, fields, models, _
from odoo.exceptions import ValidationError
```